### PR TITLE
replace functions into arrow functions

### PR DESCRIPTION
**Pull request by oprogramador**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -189,7 +189,7 @@ You can also add languages after initializing:
 const humanizer = humanizeDuration.humanizer()
 
 humanizer.languages.shortEn = {
-  y: function(c) { return c + 'y' },
+  y: () => `${c}y`,
```
Comment (EvanHahn) [inline README.md]: Does this need the `${c}` part?

Comment (oprogramador) [reply README.md]: it’s the standard way for string concarenation in es6

Comment (oprogramador) [reply README.md]: fixed

**Comment by EvanHahn** [comment]
Thanks!

**Pull request merged**